cocotb/log.py

```diff
@@ -65,89 +70,43 @@ def __init__(self, name):
         else:
             hdlr.setFormatter(SimLogFormatter())
             self.colour = False
-        self._cache = {}
-        self.name = name
-        self.handlers = []
-        self.disabled = False
-        self.filters = []
+
         self.propagate = False
-        logging.__init__(name)
```

Comment:
This line was evil, and does the equivalent of `import logging; logging.__name__ = 'something_else'`